app/app.go

```diff
@@ -1038,19 +1038,51 @@ func (app *App) LastLogs(lines int, filterLog Applog) ([]Applog, error) {
 	return logs, nil
 }
 
+type Filter struct {
+	Name      *regexp.Regexp
```

Comment:
It's safer for this field to be a string. Go regexp engine isn't completely compatible with mongodb engine, so a valid mongodb regexp might not be compilable by Go. Turning this into a string will let MongoDB be responsible for parsing and validating the regexp.